mailpoet/assets/js/src/automation/editor/components/icons/delay.tsx

```diff
@@ -0,0 +1,20 @@
+export function DelayIcon(): JSX.Element {
```

Comment:
i guess this depends on the actual implementation on how a `Step` should be rendered. I wonder if step specific files (like icons) should be organized separately. What I mean, to store them in a separate directory like `automation/steps/delay/icon.tsx`. Would help probably also with later having a workflow system with a different use case where e.g. a delay would not be needed.

Reply:
Makes sense. These things [are definitely rather mocked](https://github.com/mailpoet/mailpoet/pull/4131/commits/48fd9aa93c2bf5100a57a4a8b9b122331a0ae30d#diff-9c612c5c5cd17c514082571ad8f1a3469ddce34f3b89dd8080305bb59a90e9b6R8) for now. Maybe again something to improve on later? Should I add it to the checklist?

Reply:
sounds good :)